package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@kapeta/resource-type-rest",
-    "version": "local",
+    "version": "0.0.0-local",
```

Comment:
It has to be a semver compatible version. Chromatic was choking on some versioning logic when trying to publish storybook, so that's why this change. It's not used / gets overwritten during publish.